docs/bastion.md

```diff
@@ -0,0 +1,51 @@
+# Bastion in Kops
+
+Bastion provide an external facing point of entry into a network containing private network instances. This host can provide a single point of fortification or audit and can be started and stopped to enable or disable inbound SSH communication from the Internet, some call bastion as the "jump server".
+
+Note: Bastion will get setup for the cluster(by default) only when `--topology="private"`.
+
+* [More information on bastion from aws](http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html)
+* [More information on bastion from gce](https://cloud.google.com/solutions/connecting-securely#bastion)
+
+## AWS
+
+### Specify instance type of bastion
+
+Instance types in AWS comprise varying combinations of CPU, memory, storage, and networking capacity and give you the flexibility to choose the appropriate mix of resources for your applications.
+```
+kops create cluster --bastion-instance-type="t2.large"
+```
+
+Bastion instance type will default to `t2.medium`
+
+[More information](https://aws.amazon.com/ec2/instance-types/)
+
+
+#### Enable/disable bastion, defaults to false
+To turn on/off bastion host setup completely.
+
+|   Turn on/off Bastion   |    Example                                | Bastion ASG settings
+| ----------------------- |------------------------------------------ | --------------------
+|   Enable Bastion        |   `kops create cluster --bastion=true`    | ASG's desired/min/max set to default value 1
+|   Disable Bastion       |   `kops create cluster --bastion=false`   | ASG' desired/min/max = 0
```

Comment:
Also can we please keep our vernacular the same in our documentation? If we are using `true` / `false` in our code and in our example, we should use it in our documentation instead of `enable` / `disable`

```
Turn on/off Bastion
----------------------
 Bastion (true)
 Bastion (false)
```